assets/src/edit-story/constants/media.js

```diff
@@ -17,5 +17,14 @@
 const GB_IN_BYTES = 1024 * 1024 * 1024;
 
 export const MEDIA_TRANSCODING_MAX_FILE_SIZE = 2 * GB_IN_BYTES;
+
 export const MEDIA_POSTER_IMAGE_EXT = 'jpeg';
 export const MEDIA_POSTER_IMAGE_MIME_TYPE = 'image/jpeg';
+
+export const MEDIA_TRANSCODED_MIME_TYPE = 'video/mp4';
+export const MEDIA_TRANSCODED_FILE_TYPE = 'mp4';
```

Comment:
This should match 
```
export const MEDIA_POSTER_IMAGE_EXT = 'jpeg';
```

Either one should be change to match the other.

Reply:
You mean renaming to `MEDIA_TRANSCODED_VIDEO_MIME_TYPE`/`MEDIA_TRANSCODED_VIDEO_FILE_TYPE`? Or what exactly should match?

Reply:
The naming of `MEDIA_POSTER_IMAGE_EXT` / `MEDIA_TRANSCODED_FILE_TYPE` do not match. Where as `MEDIA_POSTER_IMAGE_MIME_TYPE` and `MEDIA_TRANSCODED_MIME_TYPE` does.

Reply:
Renaming `MEDIA_POSTER_IMAGE_EXT` to `MEDIA_POSTER_IMAGE_FILE_TYPE`